ecosystem/sep-0024.md

```diff
@@ -540,15 +540,15 @@ Name | Type | Description
 `status` | string | Processing status of deposit/withdrawal.
 `status_eta` | number | (optional) Estimated number of seconds until a status change is expected.
 `more_info_url` | string | A URL the user can visit if they want more information about their transaction's status.
-`amount_in` | string | (optional) Amount received by anchor at start of transaction as a string with up to 7 decimals. Excludes any fees charged before the anchor received the funds.
-`amount_out` | string | (optional) Amount sent by anchor to user at end of transaction as a string with up to 7 decimals. Excludes amount converted to XLM to fund account and any external fees.
-`amount_fee` | string | (optional) Amount of fee charged by anchor.
-`started_at` | UTC ISO 8601 string | (optional) Start date and time of transaction.
-`completed_at` | UTC ISO 8601 string | (optional) Completion date and time of transaction.
-`stellar_transaction_id` | string | (optional) transaction_id on Stellar network of the transfer that either completed the deposit or started the withdrawal.
+`amount_in` | string | Amount received by anchor at start of transaction as a string with up to 7 decimals. Excludes any fees charged before the anchor received the funds.
+`amount_out` | string | Amount sent by anchor to user at end of transaction as a string with up to 7 decimals. Excludes amount converted to XLM to fund account and any external fees.
+`amount_fee` | string | Amount of fee charged by anchor.
+`started_at` | UTC ISO 8601 string | Start date and time of transaction.
+`completed_at` | UTC ISO 8601 string | Completion date and time of transaction.  Return null for in-progress transactions.
+`stellar_transaction_id` | string | transaction_id on Stellar network of the transfer that either completed the deposit or started the withdrawal.
 `external_transaction_id` | string | (optional) ID of transaction on external network that either started the deposit or completed the withdrawal.
 `message` | string | (optional) Human readable explanation of transaction status, if needed.
-`refunded` | boolean | (optional) Should be true if the transaction was refunded. Not including this field means the transaction was not refunded.
+`refunded` | boolean | True if the transaction was refunded.
```

Comment:
I think it is worth calling out that this should or shouldn't be set to a value in the non-refunded case. Should it be omitted? Should it be false? Can it be either? If we make it always required and false it will be probably easier to validate implementations.
```suggestion
`refunded` | boolean | True if the transaction was refunded, false otherwise.
```

Reply:
Yeah the goal was to always have a value there, will clarify